abi/WormholeL2ERC20.sol/WormholeL2ERC20.json

```diff
@@ -0,0 +1,6006 @@
+{
```

Comment:
FYI, on my I learned that we should probably change the abi to be like in this example https://viem.sh/docs/contract/readContract.html#usage. This change to the abi should allow for more typing to be inferred. At least is that what happened on my branch

Reply:
thanks @alexkeating 

It looks like using human-readable ABI also solves the typing issue